ncl/ncl_entries/great_circle.ipynb

```diff
@@ -180,6 +188,44 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## gc_qarea\n",
+    "NCL's `gc_qarea` calculates the area of a (four-sided) quadrilateral patch on the unit sphere\n",
+    "\n",
+    "<div class=\"admonition alert alert-info\">\n",
+    "    <p class=\"admonition-title\" style=\"font-weight:bold\">Important Note</p>\n",
+    "    Coordinates should be within the valid latitude/longitude range (-90° to 90° and -180° to 180°) and be in clockwise order\n",
+    "</div>"
```

Comment:
Is this for NCL or PyProj?  If I recall correctly, PyProj has a bit more flexibility

Reply:
Ah, makes sense.  

I just had noticed that at least the valid range of longitudes with PyProj seems to be wider so wasn't sure if that came from NCL.

Not a big deal.  Just caught my eye.